src/osara.h

```diff
@@ -261,5 +261,7 @@ IReaperControlSurface* createSurface();
 extern bool selectedEnvelopeIsTake;
 // exports.cpp
 void registerExports(reaper_plugin_info_t* rec);
+// fxChain.cpp
+bool maybeOpenFxPresetDialog();
```

Comment:
Is this one really needed given src/fxChain.h is there?

Reply:
Ooo! Great catch. Nope, that was before I decided to stop being lazy and add a new header file... but then I forgot to remove the manifestation of my laziness. :)